CONTRIBUTING.md

```diff
@@ -3,4 +3,189 @@ A “Contribution” is any work that is voluntarily submitted to a project. Thi
+
+## Code of conduct
+All projects have adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Guiding principles
+FAST-DNA brings together a cohesive design vision across many web properties, providing a fast centralized location for designers, developers, and partners to learn, share, and contribute.
+
+Contributors work in concert with one another on behalf of the community of users who build web applications and sites. Contributors must demonstrate an ongoing commitment, not only to the project, but to the stability and vitality of the community as a whole. Once demonstrated, contributors may become collaborators. Before any contribution can be accepted into the project, it requires a review by existing collaborators in accordance to the established guidelines and policies. Over time, collaborators may become members of the steering committee.  The steering committee is made up of industry experts, key partners, stakeholders, and visionaries that have a common set of shared objectives and principles.
+
+It is vital that users trust the contributors to have their best interests in mind while making decisions affecting the overall direction. This means having defined guidelines, a clear roadmap, and a predictable release schedule.
+
+Adoption of new capabilities and features must be carefully balanced by the expressed needs of the users and the community. Change for the sake of change must be avoided.
+
+In order to best serve the community as a whole, the development, release, and work management processes must reflect these guiding principles.
+
+## Governance
+### Steering Committee
+The Fluent Web Steering Committee ("Steering Committee"), membership consists of key Fluent Web Collaborators ("Collaborators") who have demonstrated both design and technical expertise critical to the ongoing support, maintenance, evolution, and a long term commitment to driving the project and community forward.
+
+Collaborators can be nominated to become members of the Steering Committee following the same nomination and approval model. However, Collaborators cannot nominate themselves for Steering Committee membership. A Contributor cannot be nominated and accepted into the Steering Committee without first having been nominated and accepted as a Collaborator.
+
+### Owners
+Fluent Web Owners ("Owners"), having 'admin' access, have ultimate authority and are responsible for management, maintainenance, and operations of repositories.
```

Comment:
Should we reference FW here? Seems like this would be specific to the -msft package

Reply:
same as above